packages/@sfcc-core/core-graphql/src/index.ts

```diff
@@ -71,11 +95,14 @@ export default class CoreGraphQL {
         return this._resolvers;
     }
 
+    getExtensionFactories() {
+        return core.getExtension(API_EXTENSIONS_KEY);
+    }
+
     start() {
         logger.info('Start CoreGraphQL');
-        const expressApp = core.getService(EXPRESS_KEY);
-        const apiConfig = core.getService(API_CONFIG_KEY).config;
-        logger.info(apiConfig);
+        const expressApp = core.getService<express.Application>(EXPRESS_KEY);
+        const apiConfig = core.getService<ApiConfig>(API_CONFIG_KEY).config;
```

Comment:
Core can register many different types of services that are totally different. By passing a generic type to `getService`, we can define the type of service we are expecting back.

Reply:
We could make logger and apiconfig etc special also in core since they are now tied to it and expected: e.g. core.getApiConfig()